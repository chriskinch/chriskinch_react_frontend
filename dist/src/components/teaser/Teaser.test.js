'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _Teaser = require('./Teaser');

var _Teaser2 = _interopRequireDefault(_Teaser);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

it('Snapshot renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_Teaser2.default, { url: 'http://chriskinch.netlify.com', title: 'My teaser card', body: 'Example body txt', primaryImage: 'http://www.roshe-run.co.uk/wp-content/uploads/2018/10/Cat-Picture-4.jpg' })).toJSON();
  expect(tree).toMatchSnapshot();
});

//# sourceMappingURL=Teaser.test.js.map