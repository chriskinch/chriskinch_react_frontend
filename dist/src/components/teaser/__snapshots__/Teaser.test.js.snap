// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders correctly 1`] = `
<div
  className="views-row"
>
  <article
    className="node node--type-article node--promoted node--view-mode-teaser"
  >
    <h2
      className="node--title"
    >
      <a
        onClick={[Function]}
        style={Object {}}
      >
        My teaser card
      </a>
    </h2>
    <div
      className="field field--name-field-teaser-image field--type-image field--label-hidden field__item"
    >
      <a
        onClick={[Function]}
        style={Object {}}
      >
        <picture>
          <source
            media="(max-width: 600px)"
            srcSet="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
          />
          <img
            height={500}
            src="data:image/gif;base64,R0lGODlhAQABAIAAAMLCwgAAACH5BAAAAAAALAAAAAABAAEAAAICRAEAOw=="
            typeof="foaf:Image"
            width={500}
          />
        </picture>
      </a>
    </div>
    <ul
      className="field field--name-field-category field__items"
    />
    <div
      className="clearfix text-formatted field field--name-body field--type-text-with-summary field--label-hidden field__item quickedit-field"
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      }
    />
    <ul
      className="field field--name-field-tags field__items"
    />
  </article>
</div>
`;
